processors/zipkin/src/main/java/org/hawkular/apm/processor/zipkin/TraceCompletionTimeDeriver.java

```diff
@@ -70,9 +70,9 @@ public CompletionTime processOneToOne(String tenantId, Span item) throws RetryAt
                 ct.setEndpointType("Unknown");
             }
 
-            ct.setDuration(TimeUnit.MILLISECONDS.convert(item.getDuration(), TimeUnit.NANOSECONDS));
+            ct.setDuration(TimeUnit.MILLISECONDS.convert(item.getDuration(), TimeUnit.MICROSECONDS));
```

Comment:
we should probably consider before 1.0, but generally I don't think nano second level is useful as it is the performance issues that are of interest to users - either internally or cross boundary, so they will be higher values and therefore easy to interpret in milliseconds. The nanoseconds for internal components are being captured in the source data in case someone has a special case that requires that level of detail, in which case they can retrieve it.